examples/rpl_udp/udp.c

```diff
@@ -102,17 +102,21 @@ int udp_send(int argc, char **argv)
     ipv6_addr_t ipaddr;
     int bytes_sent;
     int address;
-    char text[5];
 
     if (argc != 3) {
         printf("usage: send <addr> <text>\n");
         return 1;
     }
 
-    address = atoi(argv[1]);
+    /* max payload size = MTU - MAC - AES - IPV6_HDR_LEN - UDP_HDR_LEN
+     *              33  = 127 - 25 - 21 - 40 - 8
```

Comment:
The minimal average would be the cc1100 with 58, but since this is smaller than 0 with above calculation, I guess we are okay :D